.github/workflows/ci.yml

```diff
@@ -45,4 +55,35 @@ jobs:
 
       - name: Format code
         run: opam exec -- dune build --auto-promote @fmt
-        if: runner.os == 'Linux'
+
+  build-macos:
```

Comment:
We can remove this if we're not running the tests on it I think, not sure this is very useful to test that it builds only?

Reply:
It is not in the last version of the code! I merge this, then. Thanks for the review :+1: